### PR TITLE
Add user-agent

### PR DESCRIPTION
**Pull request by abtris**
- add `user-agent` header
- message from apiary is print to output

```
$ ./bin/apiary publish --api-name=mynewapi1232112121 --path=./features/fixtures/apiary.apib                                                                     
Apiary service responded with an error: Suite mynewapi1232112121 does not exist
```

### CODE REVIEW

File: lib/apiary/command/preview.rb [hunk 1]
```diff
@@ -30,11 +30,15 @@ def initialize(opts)
         @options = OpenStruct.new(opts)
         @options.path         ||= 'apiary.apib'
         @options.api_host     ||= 'api.apiary.io'
-        @options.headers      ||= {:accept => 'text/html', :content_type => 'text/plain'}
         @options.port         ||= 8080
         @options.proxy        ||= ENV['http_proxy']
         @options.server       ||= false
-	@options.host	      ||= '127.0.0.1'
+	      @options.host         ||= '127.0.0.1'
```
Comment (smizell) [inline lib/apiary/command/preview.rb]: Nit: Spacing is off for me now.

File: lib/apiary/command/preview.rb [hunk 2]
```diff
@@ -30,11 +30,15 @@ def initialize(opts)
...
+        @options.headers      ||= {
+          :accept => 'text/html',
+          :content_type => 'text/plain',
```
Comment (smizell) [inline lib/apiary/command/preview.rb]: Question: should this be the content type of the document? For example, should it be the content type for API Blueprint?

Comment (smizell) [inline lib/apiary/command/preview.rb]: This question is unrelated to this PR, so it is not blocking.

Comment (abtris) [inline lib/apiary/command/preview.rb]: We are sending as plain text document. We can't have detection for swagger/api blueprint in ruby.

File: lib/apiary/command/preview.rb (hunk 1 above)
Comment (abtris) [inline lib/apiary/command/preview.rb]: yes, but in editor is ok.

image

Comment (smizell) [inline lib/apiary/command/preview.rb]: Is this because tabs/spaces are mixed?

Comment (smizell) [inline lib/apiary/command/preview.rb]: I think I see a tab on this line.

Comment (abtris) [inline lib/apiary/command/preview.rb]: Can be. I move all to spaces.

**Comment by smizell** [comment]
@abtris This looks good to go now.

**Comment by abtris** [comment]
I'll merge and release after all tests passed.
